tests/appverifier_tests/README.md

```diff
@@ -0,0 +1,29 @@
+# Thread_local Tests Suite
```

Comment:
After renaming `Thread_local` should be also aligned

Reply:
thanks, updated